src/VerifierAlgorithm.ts

```diff
@@ -155,14 +167,27 @@ export function verifyRecoverableES256K(
     const recoveredAddress: string = toEthereumAddress(recoveredPublicKeyHex).toLowerCase()
 
     const signer: VerificationMethod | undefined = authenticators.find((pk: VerificationMethod) => {
-      const keyHex = bytesToHex(extractPublicKeyBytes(pk))
-      return (
-        keyHex === recoveredPublicKeyHex ||
-        keyHex === recoveredCompressedPublicKeyHex ||
-        pk.ethereumAddress?.toLowerCase() === recoveredAddress ||
-        pk.blockchainAccountId?.split('@eip155')?.[0].toLowerCase() === recoveredAddress || // CAIP-2
-        verifyBlockchainAccountId(recoveredPublicKeyHex, pk.blockchainAccountId) // CAIP-10
-      )
+      const publickeys = extractPublicKeyBytes(pk)
+      if (Array.isArray(publickeys)) {
+        return publickeys.some((publickey) => {
```

Comment:
some will check that 1 of M passes. We need to check that N (threshold) of M keys are verified

Reply:
that gets checked in the did-jwt-vc